Python Examples/TestMonitor/test_data_manager_client.py

```diff
@@ -0,0 +1,103 @@
+import sys
+import requests
+
+create_results_host = "nitestmonitor/v2/results"
```

Comment:
The DotNet examples are able to send the request irrespective of '/' at the end of the URL, but Python example is throwing error. Can we make this example to work without considering '/' at the end?

Reply:
Applied this suggestion